modules/administration-guide/examples/snip_che-deploy-open-vsx-using-oc.adoc

```diff
@@ -0,0 +1,140 @@
+* Installed `oc` and git tools.
+* Installed Podman.
+* Log in to the OpenShift cluster where a {prod-short} is deployed as an administrator.
++
+[TIP]
+====
+
+`$ oc login pass:c,a,q[{prod-url}] --username=__<my_user>__`
+
+====
+
+.Procedure
+
+. Create a new OpenShift project for Open VSX.
++
+[subs="+attributes,+quotes"]
+----
+oc new-project openvsx
+----
+. Clone Open VSX sources.
++
+Get Open VSX sources and navigate to the OpenShift deploy directory:
++
+[bash,subs="verbatim",options="nowrap"]
+----
+git clone https://github.com/svor/openvsx.git &&
+cd openvsx &&
+git checkout sv-update-os-deployment &&
+cd deploy/openshift
+----
+
+. Prepare Open VSX server image.
++
+The next step is to build and publish the Open VSX server image. To do this, run the following commands:
++
+[bash,subs="verbatim",options="nowrap"]
+----
+export REGISTRY=quay.io &&
+export NAMESPACE=myuser &&
+export OPENVSX_SERVER_IMAGE_NAME=openvsx-server &&
+export OPENVSX_VERSION=v0.27.0 &&
+export OPENVSX_SERVER_IMAGE=${REGISTRY}/${NAMESPACE}/${OPENVSX_SERVER_IMAGE_NAME}:${OPENVSX_VERSION} &&
+podman build -t "${OPENVSX_SERVER_IMAGE}" --build-arg "OPENVSX_VERSION=${OPENVSX_VERSION}" -f openvsx.Dockerfile . &&
+podman login "${REGISTRY}" &&
+podman push "${OPENVSX_SERVER_IMAGE}"
+----
++
+[IMPORTANT]
+====
+
+Make sure to configure the correct Docker registry and namespace for your image. Additionally, ensure that the image is publicly accessible so that your OpenShift cluster can pull it without requiring authentication.
+
+====
+
+. Prepare Open VSX CLI image.
++
+Build and publish the Open VSX CLI image. To do this, run the following commands:
++
+[bash,subs="verbatim",options="nowrap"]
+----
+export OPENVSX_CLI_IMAGE_NAME=openvsx-cli
+export OPENVSX_CLI_IMAGE=${REGISTRY}/${NAMESPACE}/${OPENVSX_CLI_IMAGE_NAME} &&
+podman build -t "${OPENVSX_CLI_IMAGE}" -f cli.Dockerfile . &&
+podman push "${OPENVSX_CLI_IMAGE}"
+----
++
+[IMPORTANT]
+====
```

Comment:
wondering if we should publish an image on https://quay.io/ as well?

Reply:
Do you mean whether we need to publish the cli.Dockerfile, or are you asking if we should have prebuilt images for the OpenVSX CLI and OpenVSX available in the registry to avoid manual build/publish steps?

- Yes we need to publish images OpenVSX CLI and OpenVSX on the registry to reuse them when deploying OpneVSX
- For now, I wouldn’t publish these images ourselves (maybe need another issue for that, but for now I think we should be good with the current instructions)